libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -523,22 +526,22 @@ void NavEKF3_core::readGpsData()
     // limit update rate to avoid overflowing the FIFO buffer
     const AP_GPS &gps = AP::gps();
 
-    if (gps.last_message_time_ms() - lastTimeGpsReceived_ms > frontend->sensorIntervalMin_ms) {
-        if (gps.status() >= AP_GPS::GPS_OK_FIX_3D) {
+        if (gps.last_message_time_ms(selected_gps) - lastTimeGpsReceived_ms > 70) {
```

Comment:
cut&paste bug, in EKF3 we use frontend->sensorIntervalMin_ms

Reply:
fixed, sorry